pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -626,6 +626,62 @@
             + " non-backlog consumers as well.")
     private boolean dispatchThrottlingOnNonBacklogConsumerEnabled = false;
 
+    // <-- Pluggable dispatcher -->
```

Comment:
Do we need to add the following configurations into the `broker.conf` and `standalone.conf`?